config/src/config/indexer_table_info_config.rs

```diff
@@ -6,20 +6,33 @@ use serde::{Deserialize, Serialize};
 // Useful defaults
 pub const DEFAULT_PARSER_TASK_COUNT: u16 = 20;
 pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 1000;
+pub const DEFAULT_TABLE_INFO_BUCKET: &str = "default-table-info";
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+pub enum TableInfoServiceMode {
+    /// Backup service mode with GCS bucket name.
+    Backup(String),
+    /// Restore service mode with GCS bucket name.
+    Restore(String),
```

Comment:
how do you plan to use Restore mode?

Reply:
That's a good question.

Restore mode is for people to use when bootstrap a new node. It can help when the node lags/has issues and the rest of the world has already pruned the data. 

IndexingOnly - no backup or restore